### PR TITLE
Topic/docker testing

### PR DESCRIPTION
**Pull request by minshallj**
Hopefully @mvollrath has some comments on this. It all works just fine on my development system that has none of the ROS stack whatsoever installed.

### CODE REVIEW

**Comment by wzin** [comment]
@minshallj please add documentation on what to install and how to launch it on dev system.

**Comment by minshallj** [comment]
`./scripts/test_docker.sh` should be all that's needed :) but I'll add some documentation.

**Comment by wzin** [comment]
Kk - just create a section with docker testing with needed minimal info.

File: scripts/docker_entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+# setup ros environment
+source "/home/galadmin/catkin_ws/catkin/devel/setup.bash"
```
Comment (mvollrath) [inline scripts/docker_entrypoint.sh]: Use a path relative to this script.

**Comment by mvollrath** [comment]
When running on OS X:

```
$ scripts/test_docker.sh 
scripts/test_docker.sh: line 1: realpath: command not found
usage: dirname path
mkdir: docker_nodes/: Permission denied
cp: the -H, -L, and -P options may not be specified with the -r option.
building /-test
unable to prepare context: Cannot locate Dockerfile: absDockerfile: "/Dockerfile"
running /-test
Unable to find image '/-test:latest' locally
repository name component must match "[a-z0-9]+(?:[._-][a-z0-9]+)*"
```

**Pull request merged**